internal/util/credentials.go

```diff
@@ -33,6 +33,7 @@ const (
 	tmpKeyFileNamePrefix = "keyfile-"
 )
 
+// Credentials struct represents secret fields.
```

Comment:
```suggestion
// Credentials struct represents credentials to access the ceph cluster.
```

Reply:
Thanks for the suggestions... I'll make the changes.